### PR TITLE
Add database schema

### PR DESCRIPTION
**Pull request by teckim**
### In this pull request:

- Created tables based on the diagram.
- Added relationships between tables.
- Added join table `treatments_medical_histories` between `treatments` and `medical_histories`.

> feature implemented by: @Terbeche and @teckim

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  invoice_id INT REFERENCES invoices(id),
+  treatment_id INT REFERENCES treatments(id),
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE treatments_medical_histories(
+  treatment_id INT REFERENCES treatments(id),
+  medical_history_id INT REFERENCES medical_histories(id),
+  PRIMARY KEY(treatment_id, medical_history_id)
+);
```
Comment (ShahierNashaat) [inline schema_based_on_diagram.sql:46]: - [x] Good job creating all the tables but kindly be noticed that you should create indexes for all FOREIGN KEYS.
```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
CREATE INDEX ON invoice_items (treatment_id);
CREATE INDEX ON treatments_medical_histories (medical_history_id);
CREATE INDEX ON treatments_medical_histories (treatment_id);
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (Terbeche) [reply schema_based_on_diagram.sql:46]: Thank you

**Review by gemmen29 — APPROVED** [review]
Hi @teckim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**